source/patterns/@aws-solutions-constructs/aws-events-rule-kinesisfirehose-s3/test/integ.events-rule-kinesisfirehose-s3-no-arguments.ts

```diff
@@ -23,6 +23,12 @@ stack.templateOptions.description = 'Integration Test for aws-events-rule-kinesi
 const props: EventsRuleToKinesisFirehoseToS3Props = {
   eventRuleProps: {
     schedule: events.Schedule.rate(Duration.minutes(5))
+  },
+  bucketProps: {
+    removalPolicy: RemovalPolicy.DESTROY
+  },
+  logGroupProps: {
+    removalPolicy: RemovalPolicy.DESTROY
   }
```

Comment:
Interesting, this no arguments test leaves the logging turned on...

OTOH - we don't send logging bucket props to the logging bucket is not cleaned up as far as I can see.

Reply:
Specifying a removalPolicy for bucketProps and not loggingBucketProps will copy the same removalPolicy over to loggingBucketProps because of our condition